README.md

```diff
@@ -1,9 +1,22 @@
 # Cycle accurate benchmarking of security critical operations in hardware
 
-The benchmarking tool is separated into two rust projects:
+The top level folder structure is as follows:
 
-- `suite/` - the benchmarking suite running on the FPGA and performing the actual benchmarks
+- `benchmarks/` - collection of benchmark description files that are used to perform benchmarks
 - `cli/` - a command line interface tool that connects with the suite and controls it
+- `common/` - a crate defining used by both the cli & suite that defines the communication between them
```

Comment:
Should this just be "a crate used by both the cli and suite that defines..."?

Reply:
Yes, unfortunately I wrote the readme rather hastily, I will fix all these errors.